cli/tests/unit/test_marathon_pod.py

```diff
@@ -41,6 +43,38 @@ def test_pod_show_propagates_exceptions_from_show_pod():
         Exception('Oops!'))
 
 
+def test_pod_list_success_case():
```

Comment:
Sure, there will be more assertions in a single test method that way, but the test output makes it fairly easy to determine which one failed. I'll combine them.